tests/CMakeLists.txt

```diff
@@ -58,7 +58,7 @@ if(ENABLE_SWIFT)
                           swiftSwiftOnoneSupport)
 endif()
 
-function(add_unit_test name)
+function(add_unit_test name uses_bsd_overlay)
```

Comment:
Just make this a keyword argument (add `NO_BSD_OVERLAY` to the `options` list below), you can reference it as `AUT_NO_BSD_OVERLAY` then.